assemblies/assembly-getting-started-business-optimizer.adoc

```diff
@@ -44,30 +44,7 @@ include::{enterprise-dir}/optimizer/optimizer-running-the-examples-in-an-IDE-pro
 // include::{enterprise-dir}/optimizer/examples-table-ref.adoc[leveloffset=+2]
 
 
-include::{enterprise-dir}/employee-rostering/workbench-er-tutorial-con.adoc[leveloffset=+1]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-example-deploying-in-central-proc.adoc[leveloffset=+2]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-example-recreate-con.adoc[leveloffset=+2]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-example-setup-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-facts-vs-entities-con.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-data-model-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-planning-entity-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-planning-solution-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-constraints-con.adoc[leveloffset=+3]
-include::{drools-dir}/AuthoringAssets/drl-rules-con.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-optimizer-drools-rules-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-guided-rule-editor-proc.adoc[leveloffset=+3]
-include::{drools-dir}/AuthoringAssets/guided-rules-con.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-balance-employees-shift-rule-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-one-shift-per-day-rule-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-shift-required-skills-met-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-day-off-request-rule-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-solver-configuration-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-solver-termination-configuration-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-using-rest-proc.adoc[leveloffset=+2]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-register-solver-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-call-solver-proc.adoc[leveloffset=+3]
-
-
+include::assembly-optaplanner-employee-rostering.adoc[]
 //Getting started with quarkus
 
 include::{enterprise-dir}/optimizer/optaplanner-quarkus-con.adoc[leveloffset=+1]
```

Comment:
I am a bit confused? could you please point me out where this document could be found at
http://file.ork.redhat.com/~emmurphy/BXMSDOC-8238-pam

Reply:
@dupliaka here is the link with the anchor:
http://file.ork.redhat.com/~emmurphy/BXMSDOC-8238-pam/#assembly-optaplanner-employee-rostering_getting-started